bin/tximport.r

```diff
@@ -1,94 +1,142 @@
 #!/usr/bin/env Rscript
 
+# Script for importing and processing transcript-level quantifications.
 # Written by Lorena Pantano and released under the MIT license.
```

Comment:
I think you can add your name here given the amount of modifications

Reply:
Thanks!